src/index.js

```diff
@@ -161,6 +161,14 @@ export const transformIncludesAndExcludes = (opts) => ({
   builtIns: opts.filter((opt) => opt.match(/^(es\d+|web)\./))
 });
 
+function getPlatformSpecificDefaultFor(targets) {
+  const targetNames = Object.keys(targets);
+  const isAnyTarget = !targetNames.length;
+  const isWebTarget = targetNames.some((name) => name !== "node");
```

Comment:
If we have node + electron or node + uglify, isWebTarget will be true. But we don't need to include it in these cases ?